src/components/ebay-menu/component.js

```diff
@@ -48,7 +50,7 @@ module.exports = assign({}, menuUtils, {
     },
 
     handleItemKeypress({ key }) {
-        const itemIndex = NodeListUtils.findNodeWithFirstChar(this.getEl('menu').children, key);
+        const itemIndex = typeahead(this.getEl('menu').children, key, TIMEOUT_LENGTH);
```

Comment:
Nice clean interface. I like it.

Reply:
Just thought though, do you cover for edge cases like missing menu element or zero children?

Reply:
yes -> added tests in the makeup repo for those cases